src/GeositeFramework/js/Legend.js

```diff
@@ -66,23 +68,50 @@ define(['use!Geosite',
                     return this.tmplLegendItemScale;
                 }
             }
+
             if (legend.legend.length === 1) {
                 return this.tmplLegendItemSingle;
             }
+
             return this.tmplLegendItemMultiple;
         },
 
         render: function(legendGroups) {
             var self = this,
                 $container = $('<div>');
+
             _.each(legendGroups, function(legendGroup) {
                 var service = legendGroup.service,
                     layer = legendGroup.layer,
                     legend = legendGroup.legend,
                     tmpl = self.getLayerTemplate(legend, service, layer);
-                    $container.append(tmpl(legend));
+
+                $container.append(tmpl(legend));
             });
+
             this.$el.find('.legend-body').html($container.html());
+            this.assignLegendEvents();
+        },
+
+        assignLegendEvents: function() {
+            var self = this;
+
+            $('.expand-legend').click(function(e) {
+                self.toggleExtraLegendItems(e);
+            });
```

Comment:
Consider using a live selector and binding this in the constructor. May also want to `this.$el.find` instead of global `$`.